src/massaction_rates.jl

```diff
@@ -3,8 +3,8 @@
 # stochiometric coefficient.
 ###############################################################################
 
-@inline @fastmath function evalrxrate(speciesvec::AbstractVector{T}, rxidx::S,
```

Comment:
Here all the calculation should be with integers except the final multiplication.